bootiso

```diff
@@ -547,7 +554,7 @@ initDevicesList() {
 listDevicesTable() {
 	typeset lsblkCmd='lsblk -o NAME,MODEL,VENDOR,SIZE,TRAN,HOTPLUG,SERIAL'
 	initDevicesList
-	if [ "$disableUSBCheck" == 'false' ]; then
+	if [ "$disableUSBCheck" == 'true' ]; then
```

Comment:
Oh nice catch. 
Let's be precise and also check `if [ "$disableThumbCheck" == 'false' ]; then; echoinfo "Listing USB thumb drives (...)"`